liota/lib/utilities/utility.py

```diff
@@ -110,6 +111,28 @@ def get_linux_version():
     return platform.platform()
 
 
+def get_default_network_interface():
+    """
+    Works with Linux.
+    :return: Default Network Interface of the Edge_System
+    """
+    cmd = "route | grep '^default' | grep -o '[^ ]*$'"
+    nw_iface = str(subprocess.check_output(cmd, shell=True)).rstrip()
+    log.info("Default network interface is : {0}".format(nw_iface))
+    return nw_iface
+
+
+def get_disk_name():
+    """
+    Works with Linux.
+    :return: Disk type of the Edge_System
+    """
+    cmd = "lsblk -io KNAME,TYPE | grep 'disk' | sed -n '1p' | grep -o '^\S*'"
```

Comment:
What happens if the system has multiple disks installed on it?  I'm not using the system in question, but my laptop has two drives, and the one returned is sdb vs. sda.  Unsure if there's a good answer if there are multiple drives like this though. (just a note since I was reading the commit)